src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/UserAssignedIdentities/RemoveAzureRmUserAssignedIdentityCmdlet.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Management.Automation;
```

Comment:
Need to be able to add piping scenarios for: Get-AzureRmUserAssignedIdentity | Remove-AzureRmUserAssignedIdentity and Find-AzureRmResource | Remove-AzureRmUserAssignedIdentity.

https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell#azure-powershell-piping-scenarios